Wikipedia/Code/PlacesViewController.swift

```diff
@@ -89,6 +89,57 @@ class PlacesViewController: UIViewController, MKMapViewDelegate, UISearchBarDele
         return PlaceSearchService(dataStore: self.dataStore)
     }()
     
+    // MARK: - Accessibility
+    
+    func configurePlacesAccessibilityRotor() {
+        if #available(iOS 10.0, *) {
+            let favoritesRotor = UIAccessibilityCustomRotor(name: "Places") { predicate in
+                
+                let allArticlePlaceAnnotations = Array(self.mapView.annotations(in: self.mapView.visibleMapRect)).filter { $0 is ArticlePlace } as! [ArticlePlace]
```

Comment:
You could fix `order isn't respecting order from the filter table` by using the `articleFetchedResultsController.fetchedObjects` - however this would also include items that are offscreen.